geometry_msgs/Vector3 value
geometry_msgs/Vector3 value_dot
geometry_msgs/Vector3 value_ddot
bool[3] disable_axis